ppb_vector/vector2.py

```diff
@@ -84,7 +84,10 @@ def __rmul__(self, other: VectorLike) -> 'Vector2':
 
     def __xor__(self, other: VectorLike) -> Real:
         """
-        Computes the magnitude of the cross product
+        Computes the magnitude of the cross product.
+
+        May overflow on the product of two large vectors
+        (operands length >= 10¹⁵⁰) and produce NaN.
```

Comment:
Can we keep the unicode out of the source? I know most editors should be handle UTF-8 at this point, but.... yeah.